include/ginkgo/core/solver/batch_solver_base.hpp

```diff
@@ -289,8 +289,9 @@ class EnableBatchSolver
             auto id = Identity::create(exec, system_matrix->get_size());
             preconditioner_ = std::move(id);
         }
-        const size_type workspace_size = system_matrix->get_num_batch_items() *
-                                         (sizeof(real_type) + sizeof(int));
+        const size_type workspace_size =
+            system_matrix->get_num_batch_items() * 32;
+        // 2 * (sizeof(real_type) + sizeof(int));
```

Comment:
So `32 == 2 * (sizeof(real_type) + sizeof(int))`? But that is wrong, `2 * (sizeof(real_type) + sizeof(int)) <= 24`

Reply:
This comment was wrong. Should be fixed now. But we want to align to 8 bytes, size of `double` to ensure there is no misaligned accesses, which may happen when using the workspace for storing both 4 bytes and 8 bytes (int and double) respectively.